index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles1.css" rel="stylesheet">
+        <title>Microverse First Project</title>
+    </head>
+    <body>
+        <header  class="toolbar">
+            <a href="#" class="text1">My Logo</a>
+            <nav>
+                <button class="nav" id="menu" type="button"><img src="Union.png" alt=""></button>
+            </nav>
+        </header>
+        
+        <div class="section">
+            <div class="headline">
+                <h1>I’m Tanmay Glad to see you!</h1>
+                <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="connect">LETS CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="Vector.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Linkedinicon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(1).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(2).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(3).svg" alt=""></a></li>
+                </ul>
+            </div>
+        </div>
```

Comment:
- [ ] As suggested by the last reviewer. The container of the section is a section tag as per project requirements. I suggest that you wrap it into a section tag instead. For semantic HTML best practices. check this [page](https://www.w3schools.com/html/html5_semantic_elements.asp) for more information.